cmd/cmd-mewn.go

```diff
@@ -5,7 +5,7 @@ package cmd
+	mewn.AddAsset(".", "../runtime/js/runtime/init.js", "1f8b08000000000000ff544c4d4b0331103d3bbfe21d7783dda807418ba047c1a3e04518d3edd48dcd264b92ba4ab7ff5db26d0f2633c3e37d6945601c1f9f009f104d986642433333b8e27a463c2b7a2a42f17ee813625d145da6648e7455da6ad0c4accbbe335fb29e7f4900f4da099cfdecf228e562134d2f63885b6c42c428ab85b35b81198644555be3450c9e7cee82ffc5cdd5f5dd628892c467529ab4c257eaaccf90f42d31d9e0ef710ba589b4520485676fb335ce2649c89de0cd589710773edb5e08b3e77130d1f4d86f76becd36f8035ae3dccab45b2a55677aeeaace528d3d5d8cd6afc3d88ca5b4e1e6bf614907a23eac774e1af91942cc090f25556c7458fe050000ffffb89259798e010000")
+	mewn.AddAsset(".", "./linuxdb.yaml", "1f8b08000000000000ffec56496fdb3c10bde757ccc91f9080f657a727dd8a16cd256d0304b9f4d250d49866c545e092c4457f7ca1c5a21279b7dcb4407df2909cf738336f3822849c65c2792bd2e085d12e3903c830155497ff0044963476655a94481dba7ab33c3aa341faa509a0a9c2043e448ff2f780d609a393e5f1768333d6ec31a314d5590223ce581654d1ac03e918ad5f610db754b90446f5e51ae0e57a7b10803457e28c755601e6288b046eaa704068e7a994f028fc1cee5dc80cd0c2138ebedd4a83901941e7507b41e53d509d81b70ba09c0abd82b0c8396146cf043f8a37c26ca7d485da958b052b81b86b987b5fb86432c9301d6b93a133c1321c33a3260e7d28bebd998e9fe02754974ba99b0381d10856de952ca084f8eed65d558ad4522bb057b87663455452a4dce7e49264f870542abb40db932945fa88692efcb4747a3bfe7f08fe3e62ff22210dda87d87eb5bd5bfbadedb43aa6bb08b5a5fd6641ca672dd85958d186e79bdbb053f7f3b5752fa8b5c6c7b86b7b90b86f2a28b8fa7c37b9163a3c6dcac0f9f307e844c1fe3056749ed8ca1c24d4af2dd2abc627cb342ba13bf56c970689b3aa237c5ac2bd6ab00cb5372e465adb8384f91eb5ff720bc7ab768091b808aa5de68c117671018ae678b259d8252c725e229013cdc31e1f16284953bd76e0750fec3ae95625dee7d510427978fe5b88ed69a887ce00a42f81fad433cc8ca5b1136a7b904ef818a1b6f4c0c651f5af0fd6f0f5a9fe36854f0793f8741339b56c1e155e5a83e8fb9d65f33fe39d2f28535403b92d3d77127529c403291aefe11ff348a10b75b0880f4d9ccf2ff791ed813411a04fc6cb8f1013855adb8348f52a42fd6e9906273487850916dcc27954ffb9321f39e5088a6acad11ef9ee1ec6b0933af781dea6cc8b643f6da242cbb1f1dc559a645f69362cd1fb05d3af000000ffff5b64652c70120000")
```

Comment:
this was the "bug". running `mewn` within `wails/cmd` after updateing the yaml..
let's get the open PRs sorted and afterwards I will re-instate elementary and opensuse support..